invenio_app_ils/records_relations/views.py

```diff
@@ -214,6 +213,96 @@ def _delete_sibling_relation(self, record, relation_type, payload):
         )
         return modified_record, record, second
 
+    def _validate_sequence_creation_payload(self, record, payload):
+        """Validate the payload when creating a new sequence relation."""
+        # NOTE: check next, and previous for two way references.
```

Comment:
what do you mean with this note?

Reply:
I meant, in broken English, that `previous` and `next` pids are both required.